packages/one-app-bundler/bin/webpackCallback.js

```diff
@@ -57,6 +57,8 @@ module.exports = function getWebpackCallback(label, isModuleBuild) {
 
     // for CI/CD have webpack exit if a module isn't found (should be default behavior but isn't)
     if (stats.compilation.errors[0]) {
+      const errorMessage = stats.compilation.errors[0].message;
+      if (errorMessage) { console.log(errorMessage); }
```

Comment:
Could we update to use `console.error` (to pipe the error message to `stderr`)?
```suggestion
      if (errorMessage) { console.error(errorMessage); }
```

Reply:
thanks for the suggestion! updated

Reply:
Does this create duplicated error output for non TS projects?

Reply:
@markbello After doing some local testing, we have errors and logs in place for the 3 types errors https://v4.webpack.js.org/api/node/#error-handling that branch where you added the error is there just in case there are compilation errors and to make webpack exit when running on CI... locally it will continue running until pressing ctrl + c 

If there aren't any other webpack errors you might be hitting that if statement because your TS custom webpack config is creating compilation errors (not bundling).